app/Http/Controllers/Widgets/TopInterfacesController.php

```diff
@@ -54,11 +55,15 @@ public function getView(Request $request)
             ->select('port_id', 'device_id', 'ifName', 'ifDescr', 'ifAlias')
             ->groupBy('port_id', 'device_id', 'ifName', 'ifDescr', 'ifAlias')
             ->where('poll_time', '>', Carbon::now()->subMinutes($data['time_interval'])->timestamp)
+            ->IsActive()
```

Comment:
FYI, the case should be `->isActive()` for scopes.

Reply:
if i change it,  'scopeisActive' is not more camelcase.
if i keep it camelcase 'isActive' cannot be found.

ideas?

Reply:
changed